frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -34,15 +34,17 @@ for (const mode of modeList) {
 export default class FeedbackCodeView extends Vue {
   @Prop() language!: string;
   @Prop() value!: string;
-  @Prop({ default: () => new Map<number, FeedbackClass>() }) feedbackMap!: Map<
-    number,
-    ArenaCourseFeedback
-  >;
+  @Prop({ default: () => new Map<number, typeof FeedbackClass>() })
+  feedbackMap!: Map<number, Feedback>;
```

Comment:
ah sabes qué? con los cambios que hiciste no _tenemos_ que guardar los componentes!

```suggestion
  @Prop({ default: () => new Map<number, ArenaCourseFeedback>() })
  feedbackMap!: Map<number, ArenaCourseFeedback>;
```